addon-test-support/in-run-loop.js

```diff
@@ -0,0 +1,24 @@
+import { begin, end } from "@ember/runloop";
+
+/**
+ * Ensures that each test is executed within a RunLoop
+ *
+ * @param {object} hooks QUnit Hooks
+ */
+export default function inRunloop(hooks) {
```

Comment:
Nice!

Reply:
Yeah, I found this was necessary to make the tests behave more like a user's actual application would, where the intermediary run-loop needing to start will just happen automatically.

A nice bonus of putting this in `addon-test-support` is that people can import it and use it themselves if they want!